sphinx/api/configuration/metadata.rst

```diff
@@ -1155,9 +1153,39 @@ The following *comparison* operators are supported:
    <=    # Less than or equal to
    !=    # Not equal to
 
-Operator precedence is intended to be the same as Python. However, with the
-current choice of language engine, the ``%`` and ``//`` operators may not
-obey this - make sure you force the correct behaviour using brackets.
+The following *membership* operator is supported:
+
+.. code-block:: python
+
+   in     # contained in (True/False)
+   # e.g. "oo" in "foobar" implies True; "foo" in ["bar", "baz"] implies False
+
+   # To test for lack of membership, combine with the "not" logical operator:
+   not in # not contained in, i.e. the opposite to that described above
+
+.. tip::
+   The ``in`` operator functions as it does in Python, testing for membership
+   in a collection which can be a sequence or string. This means that you
+   must contain value(s) to test for in a list, like so:
```

Comment:
suggestion (substitute your own preferred nouns, I was just thinking about :cheese: )

Instead of "This means..."

In some cases ``item in string`` will appear to behave the same way as ``item in [list]``:
```python
>>> "cheddar" in ["cheddar", "brie"]
True
>>> "cheddar" in "roquefort cheddar"
True
```
but this is risky, because ``item in "string"`` matches **any** substring.
```python
>>> "fort cheddar" in "roquefort cheddar"
True
```
Safer, is to match a list item:
```python
# whereas searching a list will look to match an entire list item.
>>> "c" in ["roquefort", "cheddar"]
False

#